src/coreclr/tools/Common/JitInterface/CorInfoInstructionSet.cs

```diff
@@ -1899,6 +1955,18 @@ public static InstructionSet LookupPlatformIntrinsicInstructionSet(TargetArchite
                         else
                         { return InstructionSet.X64_AVX10v1_V512; }
 
+                    case "Avx10v2":
+                        if (nestedTypeName == "X64")
+                        { return InstructionSet.X64_AVX10v2_X64; }
+                        else
+                        { return InstructionSet.X64_AVX10v2; }
+
+                    case "Avx10v2_V512":
```

Comment:
This generated code checks the incorrect top-level class name. Fix is in https://github.com/dotnet/runtime/pull/109210

Reply:
Thanks for the update. I see you PR is approved. Will wait for you to merge and rebase my PR accordingly.